ynr/apps/sopn_parsing/management/commands/sopn_test_tool.py

```diff
@@ -0,0 +1,14 @@
+import requests
+from sopn_parsing.helpers.command_helpers import BaseSOPNParsingCommand
+
+
+class Command(BaseSOPNParsingCommand):
+    def handle(self, *args, **options):
+        response = requests.get(
+            "https://candidates.democracyclub.org.uk/api/next/ballots"
```

Comment:
You will want to use the `has_sopn` filter in the URL `/api/next/ballots/?has_sopn=1`